pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -297,8 +353,9 @@ func CreateGCECloud(projectID, region, zone string, managedZones []string, netwo
 	client := oauth2.NewClient(oauth2.NoContext, tokenSource)
 	// Override the transport to make it rate-limited.
 	client.Transport = &rateLimitedRoundTripper{
-		rt:      client.Transport,
-		limiter: flowcontrol.NewTokenBucketRateLimiter(10, 100), // 10 qps, 100 bucket size.
+		rt:          client.Transport,
+		getLimiter:  flowcontrol.NewTokenBucketRateLimiter(3, 50), // 3 qps, 50 bucket size.
+		postLimiter: flowcontrol.NewTokenBucketRateLimiter(3, 50), // 3 qps, 50 bucket size.
```

Comment:
I though about it and we should have burst higher than 20 (because if we send 50 requests per second in burst, 30 of them would be rejected by GCE anyway with rate-limit-exceeded).
So both values should be equal in my opinion.

The other thing is, that probably those should be higher - 6 together is definitely too small for us. It should probably sum up to 20 or almost 20 (since GCE interface is shared between all controllers).